### DISCUSSION TITLE
[ETS2 Promods] Tiles Generated v1.XX

### DISCUSSION THREAD

**Discussion by krussedull** [TwinDragon/SCS_Map_Tiles#3] [General]
hello how to get this map to work ?

**Comment by TwinDragon** [comment]
If you're wishing to self-host the map tiles and not use the GitHub in the Dashboard settings, you can follow [this writeup](https://github.com/JAGFx/ets2-dashboard-skin/blob/master/doc/MAP.md#self-hosting) on how to do so.

**Reply by krussedull** [reply]
gant get the github to work in the setting eightet

**Reply by TwinDragon** [reply]
Had to sleep on it, but I'm not sure I understand. I've also noticed JAGFx/ets2-dashboard-skin#120 when I was looking. So, if there are problems with the actual dashboard settings, then yeah that would be the more proper place to ask.

**Comment by krussedull** [comment]
hello, yes sorry for my bad typing :P and a request, is it posible to add "finnmark addon" ? a promods addon, in your hosting? when you get it to work again ? :)

**Reply by TwinDragon** [reply]
I'll think about it. I might do a more unofficial branch if I did, but right now I don't have any real plans.

There are no plans to support other map combinations either from the Dashboard project, so one would need to host their own tiles in most cases. I'm waiting for 1.44 to hit stable so I can update the current maps. Hopefully I'll not have issues, but we'll see.
